File: app/src/main/java/com/example/sportapp/data/dto/aiAnswer/MessageDto.kt
package com.example.sportapp.CleanArchitexture.data.dto.aiAnswer

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class MessageDto (
    @SerialName("user_email") val userEmail: String,
    @SerialName("message_text") val messageText: String,
    @SerialName("is_ai_answer") val isAiAnswer: Boolean,
)

File: SportAppServer/SportAppServer/SportAppServer/Models/DTOs/MessageDTO.cs
using Newtonsoft.Json;

namespace SportAppServer.Models.DTOs
{
    public class MessageDTO
    {

        [JsonProperty("user_email")]
        public string UserEmail { get; set; }

        [JsonProperty("message_text")]
        public string MessageText { get; set; }

        [JsonProperty("is_ai_answer")]
        public bool IsAiAnswer { get; set; }


        public MessageDTO() { }

        public MessageDTO(string userEmail, string messageText, bool isAiAnswer)
        {
            UserEmail = userEmail;
            MessageText = messageText;
            IsAiAnswer = isAiAnswer;
        }
    }
}
